crates/oxc_linter/src/snapshots/switch_case_braces.snap

```diff
@@ -0,0 +1,75 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: switch_case_braces
+---
+  ⚠ eslint-plugin-unicorn(switch-case-braces):  Empty switch case shouldn't have braces and not-empty case should have braces around it.
+   ╭─[switch_case_braces.tsx:1:1]
+ 1 │ switch(something) { case 1: {} case 2: {console.log('something'); break;}}
+   ·                             ──
+   ╰────
+  help: There is less visual clutter for empty cases and proper scope for non-empty cases.
+
+  ⚠ eslint-plugin-unicorn(switch-case-braces):  Empty switch case shouldn't have braces and not-empty case should have braces around it.
+   ╭─[switch_case_braces.tsx:1:1]
+ 1 │ switch(something) { case 1: case 2: console.log('something'); break;}
+   ·                             ────────────────────────────────────────
```

Comment:
Is this span correct 🤔 ?

Reply:
I pushed a little change that will better show the span, but Im not sure if what I have done is correct, so please let me know if there is some other way to better capture what rule is finding ;)